File: Develop/Backend/src/main/java/com/hientran/sohebox/sco/EnglishSCO.java
package com.hientran.sohebox.sco;

import com.fasterxml.jackson.annotation.JsonInclude;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.EqualsAndHashCode;
import lombok.NoArgsConstructor;
import lombok.ToString;

@Data
@ToString
@AllArgsConstructor
@NoArgsConstructor
@JsonInclude(JsonInclude.Include.NON_NULL)
@EqualsAndHashCode(callSuper = true)
public class EnglishSCO extends BaseSCO {
	private SearchNumberVO id;
	private SearchTextVO keyWord;
	private SearchNumberVO wordLevelId;
	private SearchNumberVO categoryId;
	private SearchNumberVO learnDayId;
	private SearchNumberVO userId;
	private SearchNumberVO vusGradeId;
}

File: Develop/Frontend/src/app/_common/_sco/englishSCO.ts
import { BaseSCO } from "./baseSCO";
import { SearchNumber, SearchText } from "./core_sco";

export class EnglishSCO extends BaseSCO {
  id: SearchNumber;
  keyWord: SearchText;
  wordLevelId: SearchNumber;
  categoryId: SearchNumber;
  learnDayId: SearchNumber;
  userId: SearchNumber;
  vusGradeId: SearchNumber;
}
